KarhooSDK/Network/APIEndpoint.swift

```diff
@@ -112,6 +113,8 @@ enum APIEndpoint {
             return "/oauth/v2/userinfo"
         case .authRefresh:
             return "/oauth/v2/token"
+        case .paymentProvider:
```

Comment:
minor comment, would it be better to be called paymentProvider**s**? as response is a list and also the API endpoint also has the `s`

Reply:
Seems to be a single Provider rather than a list.....

Reply:
That's true at the moment. Apparently the PSP would be one until perhaps they introduce loyalty?